src/js/components/DataTable/StyledDataTable.js

```diff
@@ -105,6 +105,10 @@ const StyledDataTableHeader = styled(TableHeader)`
     width: 100%;
     table-layout: fixed;
   `}
+  ${props =>
+    props.theme.dataTable &&
+    props.theme.dataTable.header &&
+    props.theme.dataTable.header.extend};
```

Comment:
Shouldn't this be applied to the `<th />` cells instead of the `<thead />` header element?

Reply:
Done.